src/components/Shared/Table.js

```diff
@@ -318,7 +318,7 @@ const Table = ({
         const isWIP = row.wip === 2
 
         return (
-          <>
+          <Box sx={{ display: 'flex', width: '100%', justifyContent: 'center'}}>
```

Comment:
Do we need this?

Reply:
this set the icons (edit or delete) in center of column cell disregarding dimensions and if one of them is hidden or no.. Now you can notice that once one of them is hidden the position is not accurate.